tests/kola/multipath/resilient/test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+## kola:
+##   # This uses appendKernelArgs and multipath, which is QEMU only
+##   platforms: qemu
+##   description: Verify multipath resiliency against multiple boot labels and no root karg.
+##   primaryDisk: ":mpath"
+##   # notice we don't add a `root=/dev/disk/by-label/dm-mpath-root` karg here
+##   appendKernelArgs: "rd.multipath=default"
+##   additionalDisks: ["1G:serial=fakeboot", "1G:mpath,wwn=11"]
+
+set -xeuo pipefail
+
+# shellcheck disable=SC1091
+. "$KOLA_EXT_DATA/commonlib.sh"
+
+# we may race with the /boot mount
+if ! is_service_active boot.mount; then
+  fatal "boot.mount failed to activate"
+fi
```

Comment:
Potentially a little more direct:

```suggestion
systemd-run --wait --property=After=boot.mount \
    echo "Waited for boot.mount"      
```